File: charArrayAndString/usingIndex.cpp
#include<iostream>
using namespace std;
int main(){
	string a = "Sakarya";
	cout << a.at(5) << "\t" << a.at(0) << endl;
	cout << (int) a.at(1) << endl;
	cout << a.substr(3) << endl;
	char name[] = "Muhammed";
	cout << name + 3 << endl;
	puts(name);
	return 0;
}

File: charArrayAndString/usingIndex.java
package springIntro;

import java.util.*;

public class usingIndex {

	public static void main(String[] args) {
		String a = "Sakarya";
		System.out.printf("%c\t%c\n", a.charAt(0), a.charAt(5));
		System.out.println((int) a.charAt(1));
		System.out.printf("\n%.3s\n", a);
		char name[] = {'M', 'u', 'h', 'a', 'm', 'm', 'e', 'd'};
		System.out.println(a.substring(3));
		for(int i = 0; i < name.length; i++) {
			System.out.printf("%c", name[i]);
		}
		System.out.println();
		for(int i = 0; i < a.length(); i++) {
			System.out.print(a.charAt(i));
		}
	}

}
